public/components/marketing-headers/4.html

```diff
@@ -0,0 +1,116 @@
+              fill="currentColor"
+            />
+          </svg>
+        </a>
+      </div>
+
+      <div class="md:flex md:items-center md:gap-12">
+        <nav aria-label="Global" class="hidden md:block">
+          <ul class="flex items-center gap-6 text-sm">
+            <li>
+              <a class="text-gray-500 transition hover:text-gray-500/75" href="#"> About </a>
+            </li>
+
+            <li>
+              <a class="text-gray-500 transition hover:text-gray-500/75" href="#"> Careers </a>
+            </li>
+
+            <li>
+              <a class="text-gray-500 transition hover:text-gray-500/75" href="#"> History </a>
+            </li>
+
+            <li>
+              <a class="text-gray-500 transition hover:text-gray-500/75" href="#"> Services </a>
+            </li>
+
+            <li>
+              <a class="text-gray-500 transition hover:text-gray-500/75" href="#"> Projects </a>
+            </li>
+
+            <li>
+              <a class="text-gray-500 transition hover:text-gray-500/75" href="#"> Blog </a>
+            </li>
+          </ul>
+        </nav>
+
+        <div class="">
+          <button
+            type="button"
+            title="Your dashboard"
```

Comment:
Let's use a `<span class="sr-only">Toggle dashboard menu</span>` within the button instead of the `title`.

Reply:
It's done, Now, span is used instead of title attribute